README.md

```diff
@@ -1,6 +1,7 @@
 ![AWS Amplify](https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png)
 
 [![discord](https://img.shields.io/discord/308323056592486420?logo=discord)](https://discord.gg/jWVbPfC)
+[![CircleCI](https://circleci.com/gh/aws-amplify/amplify-flutter/tree/master.svg?style=shield))](https://circleci.com/gh/aws-amplify/amplify-flutter/tree/master)
```

Comment:
I think there is an extra `)` in here. You can see it if you view the file here - https://github.com/aws-amplify/amplify-flutter/blob/90ec054b10803ca3a2cef186fd1bd456933ce770/README.md

I *think* this is the extra one
```suggestion
[![CircleCI](https://circleci.com/gh/aws-amplify/amplify-flutter/tree/master.svg?style=shield)](https://circleci.com/gh/aws-amplify/amplify-flutter/tree/master)
```